site/gatsby-site/src/templates/cite.js

```diff
@@ -73,6 +73,8 @@ function CitePage(props) {
 
   const sortedReports = sortIncidentsByDatePublished(incidentReports);
 
+  const metaImage = sortedReports.length > 0 ? sortedReports[0].image_url : null;
```

Comment:
Is there an incident without reports? This probably shouldn't happen 👀

Reply:
You're right. I removed the unnecessary conditional.